cirq/ops/three_qubit_gates_test.py

```diff
@@ -87,6 +92,13 @@ def test_unitary():
         [0, 0, 0, 0, 0, 0, 0, 1],
     ]), atol=1e-8)
 
+    diagonal_angles = [np.pi * i / 4 for i in range(0, 8)]
```

Comment:
This is not testing one of the angles, because the first angle equals 0.

Reply:
Changed the angles used.